src/dict_list.jl

```diff
@@ -177,17 +177,17 @@ function [`lookup_candidate`](@ref).  This is supported for `Symbol` and
 
 ## Examples
 ```julia
-julia> c = Dict(:a => [1, 2], :b => 4, :c => @onlyif(:a == 1, [10, 11]));
+julia> d = Dict(:a => [1, 2], :b => 4, :c => @onlyif(:a == 1, [10, 11]));
 
-julia> dict_list(c) # only the case of `a=1` will get key `c`
+julia> dict_list(d) # only the case of `:a == 1` will get key `:c`
```

Comment:
I am not sure that using `:a == 1` is the best description for a comment. `:a == 1` is equivalent to `false` in Julia. In reality what we want to say is that "when the key `:a` is `1`, then we get key `:c`.

Reply:
Sure, good point.